Angular 13 issues

When I try to incorporate with angular 13 project I get the following error.
[error] Error: Cannot find module '/home/atlantageek/dev/gis/ui/node_modules/angular-crud/src/crud-module/index'

However, an index.ts file is in the crud-module directory.

--- a/README.md
+++ b/README.md
@@ -27,6 +27,7 @@ Follow the steps below to see how this library works, or [watch this screencast]
    ```
    cd angular-crud
    npm install
+   npm run build
    npm pack
    ```
    

